addons/tmf_loadouts/loadouts/ussr_army_1985.hpp

```diff
@@ -384,18 +384,22 @@ class sp : g {
 	};
 };
 
-class vg : smg {
+class vg : car {
```

Comment:
Not sure about this.

Reply:
Is there a way to randomize between AKS-74 and the AKS-74U? I'll do that if that's possible.

Reply:
Just list both, they use the same ammo.